ui/app/components/mount-backend-form.ts

```diff
@@ -192,7 +192,9 @@ export default class MountBackendForm extends Component<Args> {
   }
 
   @action
-  handleIdentityTokenKeyChange(value: string[]): void {
-    this.args.mountModel.config.identityTokenKey = value[0];
+  handleIdentityTokenKeyChange(value: string[] | string): void {
+    // if array, it's coming from the search-select component, otherwise it hit the fallback component and will come in as a string.
+    this.args.mountModel.config.identityTokenKey =
+      Array.isArray(value) && value.length > 0 ? value[0] : value;
```

Comment:
if `identityTokenKey` should be a string, I'm not sure we should check the length of the array -- otherwise if the array is empty it's going to set `identityTokenKey` to an empty array which I don't think is what we want
```suggestion
      Array.isArray(value) ? value[0] : value;
```